src/adr/0009-disable-elasticsearch-dynamic-mapping.md

```diff
@@ -0,0 +1,35 @@
+# 9. Disable ElasticSearch dynamic mapping
+
+Date: 2019-10-10
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+We rely on dynamic to expose our  mail headers as a JSON map. Dynamic mapping is enabled for adding not yet encountered headers in the mapping.
+
+This causes a serie of functional issue:
+ - Maximum field count can easily be exceeded
+ - Field type 'guess' can be wrong, leading to subsequent headers omissions [1]
+ - Document indexation needs to be paused at the index level during mapping changes to avoid concurrent changes, impacting negatively performance.
+
+## Decision
+
+Rely on nested objects to represent mail headers within a mapping
+
+## Consequences
+
+The index needs to be re-created. Document reIndexation is needs.
```

Comment:
s/is needs/is needed

Reply:
Can you please stop commenting closed PRs?